.github/workflows/docker.yml

```diff
@@ -12,6 +12,7 @@ jobs:
     name: Build Docker image 
     runs-on: ubuntu-latest
     environment: scipy-dev
+    if: github.repository_owner == 'scipy' && !contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip github]')"
```

Comment:
This is missing a quotation mark at `if: "github`, they're unbalanced now so I think this will fail.